client/landing/subscriptions/button/button.tsx

```diff
@@ -4,8 +4,8 @@ import './styles.scss';
 
 type ButtonProps = React.ComponentProps< typeof BaseButton >;
 
-const Button = ( props: ButtonProps ) => {
-	return <BaseButton className="subscription-management--button" { ...props } />;
-};
+const Button = ( props: ButtonProps ) => (
```

Comment:
I wonder if wrapping the `Button` component is really that useful -- if we really need to select these buttons, couldn't we use some kind of wrapper class (e.g. available on the top-level of the subscriptions component) and then the button's build-in class?

Reply:
(Possibly the same applies to `Notice`)

Reply:
> I wonder if wrapping the Button component is really that useful -- if we really need to select these buttons, couldn't we use some kind of wrapper class (e.g. available on the top-level of the subscriptions component) and then the button's build-in class?

This PR was initially meant to only move the components into the client package. However, what you are saying does make sense. I like the idea with adding the style to some wrapper class instead. I will address those in this PR. Thanks for participating in the review. 🙂